database/insertData/_common/03_template_C_userRegistration.js

```diff
@@ -131,7 +152,254 @@ exports.queries = [
+                      title: "Village"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: { label: "Village" }
+                    }
+                    {
+                      code: "Q11District"
+                      index: 150
+                      title: "District"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: { label: "District/Province" }
+                    }
+                    {
+                      code: "S1CurrentAddress"
+                      index: 170
+                      title: "Section 1 - Current address"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: { title: "## Current address", style: "basic" }
+                    }
+                    {
+                      code: "Q12Village"
+                      index: 180
+                      title: "Village"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: { label: "Village" }
+                    }
+                    {
+                      code: "Q13District"
+                      index: 200
+                      title: "District"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: { label: "District/Province" }
+                    }
+                  ]
+                }
+              }
+              {
+                code: "S2"
+                title: "Educational information"
+                index: 1
+                templateElementsUsingId: {
+                  create: [
+                    {
+                      code: "S2Page1"
+                      index: 10
+                      title: "Section 2"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: { title: "## Education history", style: "basic" }
+                    }
+                    {
+                      code: "Q1EducationLevel"
+                      index: 20
+                      title: "Organisation Category"
+                      elementTypePluginCode: "radioChoice"
+                      category: QUESTION
+                      helpText: "TODO: This field should be consider as another  application related to applicant - and simply displayed in company license form"
+                      parameters: {
+                        label: "Education Level"
+                        options: ["Secondary", "University"]
+                        layout: "inline"
+                      }
+                    }
```

Comment:
I would think that Secondary should be a list builder as well -- you can have multiple secondary schools (could still be a single longText field in the list though maybe)

Reply:
Better to change after feedback